flow/version.py

```diff
@@ -0,0 +1,7 @@
+# Copyright (c) 2019 The Regents of the University of Michigan
+# All rights reserved.
+# This software is licensed under the BSD 3-Clause License.
+
+__all__ = ['__version__']
+
+__version__ = '0.9.0'
```

Comment:
@csadorf The version you removed from `__init__.py` is v0.8.0. This says v0.9.0. Should this be reverted to v0.8.0?
```suggestion
__version__ = '0.8.0'
```

Reply:
I just wanted to test whether you guys pay attention..., jk, I accidentally committed that after testing, sorry about that.